static-assets/components/cstudio-forms/controls/rte-tinymce5.js

```diff
@@ -15,625 +15,680 @@
+
+            switch (prop.name) {
+              case 'imageManager' :
+                this.imageManagerName = (prop.value && prop.Value != '') ? prop.value : null;
+                break;
+              case 'videoManager' :
+                this.videoManagerName = (prop.value && prop.Value != '') ? prop.value : null;
+                break;
+              case 'width' :
+                var value = isNaN(parseInt(prop.value)) ? prop.value : parseInt(prop.value);
+                this.rteWidth = (typeof prop.value == 'string' && prop.value) ? value : '100%';
+                break;
+              case 'fileManager' :
+                this.fileManagerName = (prop.value && prop.Value != '') ? prop.value : null;
+                break;
+              case 'height' :
+                this.rteHeight = (prop.value === undefined || prop.value === '') ? 300 : parseInt(prop.value, 10);
+                break;
+              case 'autoGrow' :
+                this.autoGrow = (prop.value == 'false') ? false : true;
+                break;
+              case 'maxlength' :
+                inputEl.maxlength = prop.value;
+                break;
+              case 'forcePTags' :
+                var forcePTags = (prop.value == 'false') ? false : true;
+                break;
+              case 'forceBRTags' :
+                var forceBRTags = (prop.value == 'true') ? true : false;
+                break;
+              case 'forceRootBlockPTag' :
+                var forceRootBlockPTag = (prop.value == 'false') ? false : 'p';
+                break;
+              case 'escapeScripts' :
+                var escapeScripts = (prop.value == 'true') ? true : false;
+                this.escapeScripts = escapeScripts;
+                break;
+            }
+          }
+
+          templates = rteConfig.templates && rteConfig.templates.template ? rteConfig.templates.template : null;
+
+          // https://www.tiny.cloud/docs/plugins/
+          pluginList = rteConfig.plugins;
+          pluginList = this.autoGrow ? pluginList + ' autoresize' : pluginList;
+
+          toolbarConfig1 = (rteConfig.toolbarItems1 && rteConfig.toolbarItems1.length != 0) ?
+            rteConfig.toolbarItems1 : 'bold italic | bullist numlist';
+          toolbarConfig2 = (rteConfig.toolbarItems2 && rteConfig.toolbarItems2.length != 0) ? rteConfig.toolbarItems2 : '';
+          toolbarConfig3 = (rteConfig.toolbarItems3 && rteConfig.toolbarItems3.length != 0) ? rteConfig.toolbarItems3 : '';
+          toolbarConfig4 = (rteConfig.toolbarItems4 && rteConfig.toolbarItems4.length != 0) ? rteConfig.toolbarItems4 : '';
+
+          rteStylesheets = (rteConfig.rteStylesheets && typeof rteConfig.rteStylesheets === 'object')
+            ? rteConfig.rteStylesheets.link : null;
+
+          rteStyleOverride = rteConfig.rteStyleOverride ? rteConfig.rteStyleOverride : null;
+
+          const extendedValidElements = escapeScripts ? '' : 'script[language|type|src]';
```

Comment:
Do we need to specify the attributes?

Reply:
Just tested, not necessary. Removing them